internal/output/plaintext_fns_internal_test.go

```diff
@@ -0,0 +1,64 @@
+package output
```

Comment:
These are internal tests so I didn't have to export `resource`. I could change that though.